src/index.js

```diff
@@ -18,6 +18,11 @@ const {
       getMutation: getGenerateEtchSignUrlMutation,
     },
   },
+  queries: {
+    etchPacket: {
+      getQuery: getEtchPacketQuery,
```

Comment:
100% my doing and not yours - and pretty insignificant - but I sort of don't like this naming convention I've started here and it's a little messy, too.

I feel like the actual function names should be `generateQuery` or `generateMutation`, and they should be renamed similarly here to something like `genereateEtchPacketQuery`, etc...

Do you agree? If so, would you mind changing the names around in the mutation files and here?

Reply:
Very much agreed. I'll go ahead and rename them.